### PR TITLE
Update to Aludi al-Jaladh XP

### PR DESCRIPTION
**Pull request by sLackeR-hub**
Was giving 100,000xp for keys when it should be 100.

### CODE REVIEW

**Review by ziang4891 — APPROVED** [review]

File: Database/Patches/2011-01-LearningFromExperience/9 WeenieDefaults/SQL/Creature/Human/30629 Aludi al-Jaladh.sql [hunk 1]
```diff
@@ -222,18 +222,18 @@ INSERT INTO `weenie_properties_emote_action` (`emote_Id`, `order`, `type`, `dela
...
      , (@parent_id,  2,  10 /* Tell */, 0, 1, NULL, 'The last sighting was south and west of Fort Tethana, along the western coastline of the Direlands.', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)
-     , (@parent_id,  3,  33 /* IncrementQuest */, 0, 1, NULL, 'QuestReconnaissanceAcquired0105', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 1, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL);
+     , (@parent_id,  3,  33 /* IncrementQuest */, 0, 1, NULL, 'QuestReconnaissanceAcquired0105', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL);
```
Comment (DridsGhost) [inline Database/Patches/2011-01-LearningFromExperience/9 WeenieDefaults/SQL/Creature/Human/30629 Aludi al-Jaladh.sql:225]: Is it okay that it was incrementing by 1 before the ES conversion?

File: Database/Patches/2011-01-LearningFromExperience/9 WeenieDefaults/SQL/Creature/Human/30629 Aludi al-Jaladh.sql [hunk 2]
```diff
@@ -211,7 +211,7 @@ VALUES (30629, 13 /* QuestFailure */,      1, NULL, NULL, NULL, 'QuestReconnaiss
 SET @parent_id = LAST_INSERT_ID();
 
 INSERT INTO `weenie_properties_emote_action` (`emote_Id`, `order`, `type`, `delay`, `extent`, `motion`, `message`, `test_String`, `min`, `max`, `min_64`, `max_64`, `min_Dbl`, `max_Dbl`, `stat`, `display`, `amount`, `amount_64`, `hero_X_P_64`, `percent`, `spell_Id`, `wealth_Rating`, `treasure_Class`, `treasure_Type`, `p_Script`, `sound`, `destination_Type`, `weenie_Class_Id`, `stack_Size`, `palette`, `shade`, `try_To_Bond`, `obj_Cell_Id`, `origin_X`, `origin_Y`, `origin_Z`, `angles_W`, `angles_X`, `angles_Y`, `angles_Z`)
-VALUES (@parent_id,  0,  30 /* InqQuestSolves */, 0, 1, NULL, 'QuestReconnaissanceAcquired0105@check', NULL, 1, 1, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL);
+VALUES (@parent_id,  0,  30 /* InqQuestSolves */, 0, 1, NULL, 'QuestReconnaissanceAcquired0105@check', NULL, 1, 2147483647, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL);
```
Comment (DridsGhost) [inline Database/Patches/2011-01-LearningFromExperience/9 WeenieDefaults/SQL/Creature/Human/30629 Aludi al-Jaladh.sql:214]: Should the inqQuestSolves be 1 -1 like the original emote before conversion?

Comment (ziang4891) [reply Database/Patches/2011-01-LearningFromExperience/9 WeenieDefaults/SQL/Creature/Human/30629 Aludi al-Jaladh.sql:214]: Good catch

File: Database/Patches/2011-01-LearningFromExperience/9 WeenieDefaults/SQL/Creature/Human/30629 Aludi al-Jaladh.sql (hunk 1 above)
Comment (ziang4891) [reply Database/Patches/2011-01-LearningFromExperience/9 WeenieDefaults/SQL/Creature/Human/30629 Aludi al-Jaladh.sql:225]: good catch

**Review by ziang4891 — CHANGES_REQUESTED** [review]
Unintentional changes made, mentioned by Drid, should be reverted

File: Database/Patches/2011-01-LearningFromExperience/9 WeenieDefaults/SQL/Creature/Human/30629 Aludi al-Jaladh.sql (hunk 1 above)
Comment (sLackeR-hub) [reply Database/Patches/2011-01-LearningFromExperience/9 WeenieDefaults/SQL/Creature/Human/30629 Aludi al-Jaladh.sql]: Not sure what caused this in the .es so changed manually in the sql.

File: Database/Patches/2011-01-LearningFromExperience/9 WeenieDefaults/SQL/Creature/Human/30629 Aludi al-Jaladh.sql (hunk 2 above)
Comment (sLackeR-hub) [reply Database/Patches/2011-01-LearningFromExperience/9 WeenieDefaults/SQL/Creature/Human/30629 Aludi al-Jaladh.sql]: Not sure what caused this in the .es so changed manually in the sql.

**Review by DridsGhost — APPROVED** [review]

**Review by ziang4891 — APPROVED** [review]

**Pull request merged**